crates/sui-core/src/unit_tests/authority_tests.rs

```diff
@@ -1558,8 +1570,8 @@ async fn test_handle_sponsored_transaction() {
             budget: TEST_ONLY_GAS_UNIT_FOR_TRANSFER * rgp,
         },
     );
-    let dual_signed_tx =
-        to_sender_signed_transaction_with_multi_signers(data, vec![&sender_key, &sponsor_key]);
+    let dual_signed_tx = to_sender_signed_transaction_with_multi_signers(data, vec![&sender_key]);
+    let dual_signed_tx = authority_state.verify_transaction(dual_signed_tx).unwrap();
```

Comment:
to make this test useful, let's use `VerifiedTransaction::new_unchecked` to force the type right